### PR TITLE
fix: oxorio audit findings fixes

### PR DESCRIPTION
**Pull request by 0xmoebius**

### CODE REVIEW

File: packages/contracts/src/contracts/Entrypoint.sol [hunk 1]
```diff
@@ -37,10 +37,8 @@ contract Entrypoint is AccessControlUpgradeable, UUPSUpgradeable, ReentrancyGuar
   using SafeERC20 for IERC20;
   using ProofLib for ProofLib.WithdrawProof;
 
-  // @notice keccak256('OWNER_ROLE')
-  bytes32 internal constant _OWNER_ROLE = 0x6270edb7c868f86fda4adedba75108201087268ea345934db8bad688e1feb91b;
-  // @notice keccak256('ASP_POSTMAN')
-  bytes32 internal constant _ASP_POSTMAN = 0xfc84ade01695dae2ade01aa4226dc40bdceaf9d5dbd3bf8630b1dd5af195bbc5;
+  bytes32 internal constant _OWNER_ROLE = keccak256('OWNER_ROLE');
+  bytes32 internal constant _ASP_POSTMAN = keccak256('ASP_POSTMAN');
```
Comment (wei3erHase) [inline packages/contracts/src/contracts/Entrypoint.sol:41]: add comment with the actual value?
i would have dismissed this one tbh, but well

**Review by wei3erHase — APPROVED** [review]
looks good 👍

File: packages/contracts/src/contracts/Entrypoint.sol (hunk 1 above)
Comment (0xmoebius) [reply packages/contracts/src/contracts/Entrypoint.sol]: added at [36b5241](https://github.com/0xbow-io/privacy-pools-core/pull/33/commits/36b5241508bc381b32fb1aeaa9b7692bc123ca09)

Comment (0xng) [reply packages/contracts/src/contracts/Entrypoint.sol]: I think the previous way you had it is more common convention. For example, solady: https://github.com/Vectorized/solady/blob/main/src/tokens/ERC20.sol#L64

File: packages/contracts/src/contracts/Entrypoint.sol [hunk 2]
```diff
@@ -356,7 +372,7 @@ contract Entrypoint is AccessControlUpgradeable, UUPSUpgradeable, ReentrancyGuar
    * @param _feeBPS The fee in basis points
    */
   function _deductFee(uint256 _amount, uint256 _feeBPS) internal pure returns (uint256 _afterFees) {
-    _afterFees = _amount - (_amount * _feeBPS / 10_000);
+    _afterFees = _amount - ((_amount * _feeBPS) / 10_000);
```
Comment (0xng) [inline packages/contracts/src/contracts/Entrypoint.sol:370]: is this necessary or just for clarity?

File: packages/contracts/src/contracts/Entrypoint.sol [hunk 3]
```diff
@@ -37,10 +37,10 @@ contract Entrypoint is AccessControlUpgradeable, UUPSUpgradeable, ReentrancyGuar
...
   using ProofLib for ProofLib.WithdrawProof;
 
-  // @notice keccak256('OWNER_ROLE')
-  bytes32 internal constant _OWNER_ROLE = 0x6270edb7c868f86fda4adedba75108201087268ea345934db8bad688e1feb91b;
-  // @notice keccak256('ASP_POSTMAN')
-  bytes32 internal constant _ASP_POSTMAN = 0xfc84ade01695dae2ade01aa4226dc40bdceaf9d5dbd3bf8630b1dd5af195bbc5;
+  // @dev 0xb19546dff01e856fb3f010c267a7b1c60363cf8a4664e21cc89c26224620214e
+  bytes32 internal constant _OWNER_ROLE = keccak256('OWNER_ROLE');
+  // @dev 0xfc84ade01695dae2ade01aa4226dc40bdceaf9d5dbd3bf8630b1dd5af195bbc5
+  bytes32 internal constant _ASP_POSTMAN = keccak256('ASP_POSTMAN');
```
Comment (simon-something) [inline packages/contracts/src/contracts/Entrypoint.sol:43]: natspec should have 3 `/` iirc (or will be ignored, then remove the @ if it's intended)

Comment (0xmoebius) [reply packages/contracts/src/contracts/Entrypoint.sol]: fixed at [fix: proper natspec slashes amount](https://github.com/0xbow-io/privacy-pools-core/pull/33/commits/6cdeeaf0e44b98d5563c9d2f340f5825bec117f2)

File: packages/contracts/test/invariants/fuzz/helpers/GhostStorage.sol [hunk 4]
```diff
@@ -4,7 +4,7 @@ pragma solidity ^0.8.0;
 contract GhostStorage {
   uint256 constant FEE_DENOMINATOR = 10_000;
   uint256 constant FEE_VETTING = 101;
-  uint256 constant MAX_RELAY_FEE = 101;
+  uint256 constant MAX_RELAY_FEE = 150;
```
Comment (0xng) [inline packages/contracts/test/invariants/fuzz/helpers/GhostStorage.sol:7]: any reason for this change? just for it to be different from fee_vetting?

**Review by 0xng — APPROVED** [review]
looks good! just for confirmation, the one where the vetting fee is checked on every deposit for correctness has been chosen to be ignored due to the extra gas costs/worse ux/being too edge case?

**Review by 0xtiti — APPROVED** [review]

File: packages/contracts/test/unit/core/Entrypoint.t.sol [hunk 5]
```diff
@@ -276,13 +284,19 @@ contract UnitDeposit is UnitEntrypoint {
   )
     external
     givenPoolExists(
-      PoolParams({pool: _params.pool, asset: _ETH, minDeposit: _params.minDeposit, vettingFeeBPS: _params.vettingFeeBPS})
+      PoolParams({
+        pool: _params.pool,
+        asset: _ETH,
+        minDeposit: _params.minDeposit,
+        vettingFeeBPS: _params.vettingFeeBPS,
+        maxRelayFeeBPS: 500 // Default to 5%
```
Comment (wei3erHase) [inline packages/contracts/test/unit/core/Entrypoint.t.sol:292]: why not params here?
